buildSrc/src/main/groovy/com/github/jk1/license/render/CsvCustomReportRenderer.groovy

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.jk1.license.render
```

Comment:
Why do you use a different package from ours? Are there any package-private fields/methods?
If so, please, leave a note about that in the class descritpion

Reply:
Sorry, It was a test. I'm not going to add custom render class into repository